.buildkite/danger/Dangerfile

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+def release_branch?
+  danger.github.branch_for_base.start_with?('release/') || danger.github.branch_for_base.start_with?('hotfix/')
+end
+
+def main_branch?
+  danger.github.branch_for_base == 'trunk'
+end
+
+def wip_feature?
+  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
+  has_wip_title = github.pr_title.include?('WIP')
+
+  has_wip_label || has_wip_title
+end
+
+return if github.pr_labels.include?('Releases')
+
+github.dismiss_out_of_range_messages
+
+manifest_pr_checker.check_all_manifest_lock_updated
+
+labels_checker.check(
+  do_not_merge_labels: ['status: do not merge'],
+  required_labels: [//],
+  required_labels_error: 'PR requires at least one label.'
+)
+
+view_changes_need_screenshots.view_changes_need_screenshots
+
+pr_size_checker.check_diff_size
+
+# skip for draft PRs and for WIP features unless the PR is against "develop" or "release/x.x" branches
```

Comment:
`develop` jumped to my eyes as out of place. What do you think of removing the exact branch name and defer to the "release" and "main" abstractions that have been established above?

```suggestion
# skip for draft PRs and for WIP features, unless the PR is against the main or release branch
```

Reply:
Noted, will also fix on the other apps, thanks. 7fd7aea1ad89d7e05305f5a810225491e5be7f6c (also moving `Dangerfile` to root)